scripts/create_ocd_ids.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import print_function
+import csv
+"""
+Program to generate OCD IDs for the 2019 Indian Vidhan Sabha elections.
+"""
+
+contests = {"hr": "Haryana", "mh": "Maharashtra"}
+columns = ["id", "name"]
+country = "in"
+election = "Vidhan Sabha"
```

Comment:
OCD-IDs should be independent of any one election.

Reply:
Looking at the OCD-IDs for the Lok Sabha elections, it looks like the name of the election was included in the file containing the election (I was pattern matching). I'll take this out and generalize this script better.